geoapi/services/features.py

```diff
@@ -18,55 +19,14 @@
 from geoapi.exceptions import InvalidGeoJSON, ApiException
 from geoapi.utils.assets import make_project_asset_dir, delete_assets, get_asset_relative_path
 from geoapi.log import logging
-from geoapi.utils import geometries
+from geoapi.utils import (geometries,
+                          features as features_util)
```

Comment:
nit picking: Why `features_util` as the name? In the context of where it is being used (line 355 below) it does not read as a utility, but as a list of appended `questionnaire_features`.